icons/circle/48/steam_icon_344850.svg

```diff
@@ -0,0 +1,23 @@
+<svg version="1.1" viewBox="0 0 48 48" xmlns="http://www.w3.org/2000/svg">
+ <defs>
+  <linearGradient id="linearGradient3764-6" x1="1" x2="47" gradientTransform="rotate(-90 24 24)" gradientUnits="userSpaceOnUse">
+   <stop stop-color="#6bd6ff" offset="0"/>
+   <stop stop-color="#7fdcff" offset="1"/>
+  </linearGradient>
+ </defs>
+ <path d="m36.31 5c5.859 4.062 9.688 10.831 9.688 18.5 0 12.426-10.07 22.5-22.5 22.5-7.669 0-14.438-3.828-18.5-9.688 1.037 1.822 2.306 3.499 3.781 4.969 4.085 3.712 9.514 5.969 15.469 5.969 12.703 0 23-10.298 23-23 0-5.954-2.256-11.384-5.969-15.469-1.469-1.475-3.147-2.744-4.969-3.781zm4.969 3.781c3.854 4.113 6.219 9.637 6.219 15.719 0 12.703-10.297 23-23 23-6.081 0-11.606-2.364-15.719-6.219 4.16 4.144 9.883 6.719 16.219 6.719 12.703 0 23-10.298 23-23 0-6.335-2.575-12.06-6.719-16.219z" opacity=".05"/>
+ <path d="m41.28 8.781c3.712 4.085 5.969 9.514 5.969 15.469 0 12.703-10.297 23-23 23-5.954 0-11.384-2.256-15.469-5.969 4.113 3.854 9.637 6.219 15.719 6.219 12.703 0 23-10.298 23-23 0-6.081-2.364-11.606-6.219-15.719z" opacity=".1"/>
+ <path d="m31.25 2.375c8.615 3.154 14.75 11.417 14.75 21.13 0 12.426-10.07 22.5-22.5 22.5-9.708 0-17.971-6.135-21.12-14.75a23 23 0 0 0 44.875 -7 23 23 0 0 0 -16 -21.875z" opacity=".2"/>
+ <path d="m24 1c12.703 0 23 10.297 23 23s-10.297 23-23 23-23-10.297-23-23 10.297-23 23-23z" fill="url(#linearGradient3764-6)"/>
+ <path d="m40.03 7.531c3.712 4.084 5.969 9.514 5.969 15.469 0 12.703-10.297 23-23 23-5.954 0-11.384-2.256-15.469-5.969 4.178 4.291 10.01 6.969 16.469 6.969 12.703 0 23-10.298 23-23 0-6.462-2.677-12.291-6.969-16.469z" opacity=".1"/>
+ <g transform="translate(-2,-54)">
+  <g transform="translate(1,1)"></g>
+ </g>
+ <path d="m15.75 30" fill="none"/>
+ <path d="m20.25 30" fill="none"/>
+ <g fill-rule="evenodd">
+  <path d="m20 10.586l-1.4141 1.4141 0.70703 0.70703 1.707 1.707v7.4121l-5.4199 15.174h18.84l-5.4199-15.174v-7.4121l1.707-1.707 0.70703-0.70703-1.4141-1.4141-0.70703 0.70703-2.293 2.293v8.5879l4.582 12.826h-13.164l4.582-12.826v-8.5879l-2.293-2.293-0.70703-0.70703zm3.8574 14.414l-2.8574 8h8l-2.8574-8h-2.2852z" color="#000000" color-rendering="auto" dominant-baseline="auto" image-rendering="auto" opacity=".1" shape-rendering="auto" solid-color="#000000" style="font-feature-settings:normal;font-variant-alternates:normal;font-variant-caps:normal;font-variant-ligatures:normal;font-variant-numeric:normal;font-variant-position:normal;isolation:auto;mix-blend-mode:normal;shape-padding:0;text-decoration-color:#000000;text-decoration-line:none;text-decoration-style:solid;text-indent:0;text-orientation:mixed;text-transform:none;white-space:normal"/>
```

Comment:
Tried just now, didn't change that section... I'll see if there's anything weird going on that I can find.